test/controllers/test_artefacts_controller.rb

```diff
@@ -30,6 +30,50 @@ def self._create_onts
         @@pagesize = 1
     end
 
+    def test_home_controller
+        get "/"
+        assert last_response.ok?
+        catalog_data = MultiJson.load(last_response.body)
+        expected_data = {
+            "acronym" => "OntoPortal",
```

Comment:
here you need to test the properties used for federation,

Reply:
Yes i thought about this but we need to merge this PR first to have the federation attributes in the root route: https://github.com/ontoportal-lirmm/ontologies_linked_data/pull/207

Reply:
still need to be done, now that the other is merged.

Reply:
@syphax-bouazzouni  could you review this please to merge it: https://github.com/ontoportal-lirmm/ontologies_linked_data/pull/207